README.md

```diff
@@ -2,33 +2,35 @@
 # How to Update the [MHKiT Website](https://MHKiT-Software.github.io/MHKiT/)
 
 ## Download/Install Required Packages
-### Step 1. Download and Install Python 
+### Step 1. Download and Install Python
   - Windows: Download and install Python using [Anaconda](https://www.anaconda.com/distribution/)
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already installed and added to path
- 
+
 ### Step 2. Add Python to System Path
-  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts 
+  - Windows: Modify the PATH in environmental variables to include: C:\PYTHON;C:\PYTHON\Scripts
   ([more info here](http://stackoverflow.com/questions/3701646/how-to-add-to-the-pythonpath-in-windows-7))
-  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path 
+  - **NOTE:** Anaconda skip to Step 3,  Python is already added to path
   - **NOTE:** MAC/LINUX skip to Step 3, Python is already added to path
-  
+
 ### Step 3. Download/Install [Sphinx](http://www.sphinx-doc.org/en/stable/index.html) package
-  - Install Sphinx from cmd ``pip install -U Sphinx`` 
+  - Install Sphinx from cmd ``pip install -U Sphinx``
     ([more info here](http://www.sphinx-doc.org/en/master/usage/installation.html))
   - **NOTE:** You may need to add PROXY settings ([see info here](https://cinhtau.net/2018/04/16/python-proxy-windows/))
 
 ### Step 4. Download/Install Sphinx extensions
   - Install BibTex ``pip install -U sphinxcontrib-bibtex``
   - Install MATLAB theme ``pip install -U sphinxcontrib-matlabdomain``
   - Install rtd theme ``pip install -U sphinx_rtd_theme``
+  - Install NB Sphinx ``pip install -U nbsphinx``
+  - Install NB Sphinx Link ``pip install -U nbsphinx_link``
   - Install Goggle Analytics ``pip install -U sphinxcontrib-googleanalytics``
   ([more info on Google Analytics here](https://pypi.org/project/sphinxcontrib-googleanalytics/))
   - **NOTE:** You may have to manually move extensions to the ``/sphinx/`` or ``/sphinxcontrib/`` Python directories
   - **NOTE:** For Sphinx 1.8 you may need to modify googleanalytics.py according to ([this](https://jiangsheng.net/2019/01/05/fix-sphinxcontrib-googleanalytics-on-sphinx-1-8/))
```

Comment:
@kaklise When is it that we expect to need to do this? I have not needed to for my Linux or Windows installs. Sean also said he did not need to do this.

Reply:
@ssolson this is probably not necessary once the path is set properly. Many of these the steps are probably no longer necessary since this README was originally written in ~2015 for WEC-Sim.

Reply:
@kaklise @ssolson the `googleanalytics.py` note is about how to set up Google Analytics for the documentation site. Once it's done, it never need to be modified again. However, this may no longer be true since I believe Google Analytics is included in the Sphinx distribution and can be configured in the `conf.py` file directly.